### PR TITLE
about page + duplicate phishing deleted

### PR DESCRIPTION
**Pull request by roopeky**

### CODE REVIEW

File: packages/app/src/components/About.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                    <StyleSpan>Roope Kylli</StyleSpan>
+                    <a href="https://www.instagram.com/roopek_">
+                        <img width="20" height="20" src="https://img.icons8.com/ios/50/instagram-new--v1.png" alt="instagram-new--v1"/>
+                    </a>
+                    <a href="https://github.com/roopeky">
+                        <img width="20" height="20" src="https://img.icons8.com/material-outlined/24/github.png" alt="github"/>
+                    </a>
+                </div>
+            </ul>
+            <p>Catchmaster was made in Web Development 2 course at Metropolia University by Group 2</p>
```
Comment (adnanavni) [inline packages/app/src/components/About.tsx:71]: METROPOLIA UNIVERSITY OF APPLIED SCIENCES

**Review by adnanavni — CHANGES_REQUESTED** [review]

File: packages/app/src/components/About.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    display: flex;
+    flex-direction: column;
+    background-color: #48412f;
+    text-align: center;
+    padding: 2rem;
+    height: 40rem;
+`;
+
+const StyleSpan = styled.span`
+    margin-right: 10px;
```
Comment (pennane) [inline packages/app/src/components/About.tsx:13]:
```suggestion
    margin-right: 0.625rem;
```

Comment (pennane) [reply packages/app/src/components/About.tsx:13]: px bad rem good, rem with 0.25 inverval best (i.e. 0.25rem, 0.5rem, 0.75rem)

File: packages/app/src/components/About.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            Whether you've reeled in a trophy-sized bass, a colorful trout, or a feisty marlin, Catchmaster is your platform to showcase your prized catches. 
+            With a photo and a few words to capture the moment, you can connect with like-minded individuals, learn new techniques, and celebrate the thrill of the catch together. 
+            Join us at Catchmaster, just log in easily with your Google account and start sharing your fishing adventures today!
+            </p>
+            <p>Our group is composed of the following members:</p>
+            <ul>
+            <div>
+                    <StyleSpan>Adnan Avni</StyleSpan>
+                    <a href="https://www.instagram.com/adnanavni">
+                        <img width="20" height="20" src="https://img.icons8.com/ios/50/instagram-new--v1.png" alt="instagram-new--v1"/>
```
Comment (pennane) [inline packages/app/src/components/About.tsx:34]:
```suggestion
                        <StyledAuthorImage src="https://img.icons8.com/ios/50/instagram-new--v1.png" alt="instagram-new--v1"/>
```
maybe

File: packages/app/src/components/About.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components";
+
+const StyleBox = styled.div`
```
Comment (pennane) [inline packages/app/src/components/About.tsx:3]:
```suggestion
const StyledBox = styled.div`
```

File: packages/app/src/components/About.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    display: flex;
+    flex-direction: column;
+    background-color: #48412f;
+    text-align: center;
+    padding: 2rem;
+    height: 40rem;
+`;
+
+const StyleSpan = styled.span`
```
Comment (pennane) [inline packages/app/src/components/About.tsx:12]:
```suggestion
const StyledSpan = styled.span`
```

File: packages/app/src/components/About.tsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    return (
+        <StyleBox>
+            <h1>About</h1>
+            <p>
+            Welcome to Catchmaster, your ultimate fishing companion!
+            We understand that fishing isn't just a hobby; it's a passion, a way of life. 
+            Our app is designed to unite anglers from all walks of life, creating a vibrant community where you can share your fishing triumphs with fellow enthusiasts. 
+            Whether you've reeled in a trophy-sized bass, a colorful trout, or a feisty marlin, Catchmaster is your platform to showcase your prized catches. 
+            With a photo and a few words to capture the moment, you can connect with like-minded individuals, learn new techniques, and celebrate the thrill of the catch together. 
+            Join us at Catchmaster, just log in easily with your Google account and start sharing your fishing adventures today!
```
Comment (pennane) [inline packages/app/src/components/About.tsx:27]: could maybe separate into multiple paragraphs so it looks automatically good in view

File: packages/app/src/components/About.tsx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+            </p>
+            <p>Our group is composed of the following members:</p>
+            <ul>
+            <div>
```
Comment (pennane) [inline packages/app/src/components/About.tsx:31]:
```suggestion
            <ul>
```
afaik only `<li>` elements should be direct children of a `<ul>` element

File: packages/app/src/components/About.tsx (hunk 3 above)
Comment (pennane) [reply packages/app/src/components/About.tsx:34]: Actually there is already the <Image> component that you guys have implemented before, that could possibly be re-used here.

**Review by adnanavni — APPROVED** [review]

**Pull request merged**